### DISCUSSION TITLE
How to achieve fuzzy completion

### DISCUSSION THREAD

**Discussion by jenia** [minad/vertico#471] [Q&A]
Hello.

I want to use fuzzy complete with Vertico. I installed Orderless but it's not working as expected.

Specifically, I want to be able to press M-x and type `org-calendar` and Vertico/Orderless to propose the command `org-goto-calendar` because `org-*calendar` are in the string but I can't seem to be able to achieve this.

Can someone please suggest a solution?

Thanks kindly in advance.

**Comment by jenia** [comment]
Oh, I think I can literally type regex in Vertico/Orderless :) Is that how it's meant to be used?

**Reply by Bujiraso** [reply]
What's interesting is that I can type regexes, and it'll filter, but it doesn't actually let me work with things fuzzily.

Here, try this with Vertico + Orderless

For a test.org like this
```
* Foo
** Bar
*** Baz
** Barry
* Goo
```

Type `C-u C-c C-j` in order to be prompted with a mini-buffer for a search.
Try typing `.*Barry` and you'll notice that it definitely is smart enough to filter... however you can't actually select Barry. You can hit TAB and then it'll expand the node once but you lose your search context (have to type it again).

I also can't seem to do a partial where I type `Barry` and get Barry, it still just shows Foo

Picture below:
![image](https://github.com/minad/vertico/assets/1975771/a01108b6-cd26-4a55-9cbf-3e507396c81e)
